docker-compose-test.yml

```diff
@@ -0,0 +1,79 @@
+version: "2.1"
+services:
+  consumer-test:
+    extends:
+      file: docker-compose-base.yml
+      service: ckan-consumer-base
+    environment:
+      TENANCY_HEADER: x-oauth-realm
+
+      CONSUMER_NAME: 'CKAN-TEST'
+
+      EXPOSE_PORT: 9013
+      LOG_LEVEL: "DEBUG"
+      CKAN_CONSUMER_CONFIG_PATH: "/code/tests/conf/consumer.json"
+      CKAN_CONSUMER_KAFKA_CONFIG_PATH: "/code/tests/conf/kafka.json"
+      CONNECT_RETRY_WAIT: 1
+      STARTUP_CONNECTION_RETRY: 3
+
+      # redis
+      REDIS_DB: 2
+      REDIS_HOST: redis-test
+      REDIS_PORT: 6379
+      REDIS_PASSWORD: testpassword
+
+      KAFKA_URL: kafka-test:29092
+      SECURITY.PROTOCOL: SASL_SSL
+      SASL.MECHANISM: PLAIN
+      SASL.USERNAME: root
+      SASL.PASSWORD: 09049a938686107c15ed6748010e2f1b
+
+  ckan-test:
+    image: ehealthafrica/ckan:2.8.0-rc
```

Comment:
are you using the uploaded image or the one you are building? :thinking:

Reply:
uploaded

Reply:
sorry, it was confusing, you were removing the files, not adding them :face_with_head_bandage: